File: valiha-hotel-admin/src/domain/entities/location.ts
import { Car } from "./car";
import { Client } from "./client";
import { Payment } from "./payment";

export interface Location {
  id: string;
  state: string;
  start: string;
  end: string;
  destination: string;
  reason: string;
  client: Client;
  cars: Car[];
  payment: Payment;
}

File: valihahotel/notifications/src/main/java/com/valiha/notification/domain/Location.java
package com.valiha.notification.domain;

import java.util.List;
import lombok.Data;

@Data
public class Location {

  private String id;
  private String state;
  private String start;
  private String end;
  private String destination;
  private String reason;
  private Client client;
  private List<Car> cars;
  private Payment payment;
}
